catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/impl/StorableResourceImpl.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.impl;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.apache.commons.io.IOUtils;
+import org.codice.ddf.catalog.ui.metacard.internal.StorableResource;
+import org.codice.ddf.platform.util.TemporaryFileBackedOutputStream;
+
+public class StorableResourceImpl implements StorableResource {
+
+  private TemporaryFileBackedOutputStream temporaryFileBackedOutputStream;
+  private String mimeType;
+  private String filename;
+  private String errorMessage;
+  private List<InputStream> openInputStreams = new LinkedList<>();
+
+  private StorableResourceImpl(@Nullable String mimeType, String filename) {
+    temporaryFileBackedOutputStream = new TemporaryFileBackedOutputStream();
+    this.mimeType = mimeType;
+    this.filename = filename;
+  }
+
+  /** The caller is responsible for the closing the InputStream argument. */
```

Comment:
`/** The caller is responsible for closing the InputStream argument. */`

Reply:
fixed